articles/automatic-software-install-in-fleet.md

```diff
@@ -0,0 +1,146 @@
+# Automatic installation of software on hosts
+
+![Automatic installation of software on hosts](../website/assets/images/articles/automatic-software-install-in-fleet-731x738@2x.png)
+
+Fleet has the ability to automatically and remotly install software on hosts based on labels applied to them. 
+This guide will walk you through the process of configuring fleet for automatic installation of
+software on specific hosts using pre uploded installation images and using label based install
+policy. You'll learn how to configure and use this feature, as well as understand how the underlying
+mechanism works.
+
+Fleet allows its users to upload trusted software installation file to be installed and used on hosts.
+When adding a new software, it is possible to configure this specific software to be installed to
+all hosts or alternativly to host with a specific label. 
+
+How to do it:
+- In Fleet click the upper menu `Software` tab.
+- On top left corner drop down menu, select a team (or `No team`)
+- On top right corner click `Add software`.
+- In the newly openned modal add a new trusted binary.
+- In the `Install` dropdown menu select "Automatic".
+- Under the `Target` section select `All hosts` or `Custom`.
+- If `Custom` was selected, right below it will appear a dropdown that will allow `Include` or `Exclude` specific labels. A list of labels will be shown to configure the proper operation
+
+
+In the example below Install on all hosts EXCEPT those with 'Okta - Engineering' label
+
+![Install on all hosts EXCEPT those with 'Okta - Engineering' label](../website/assets/images/articles/automatic-software-install-in-fleet-729x924@2x.png)
+
+
+Current supported installation files, manual upload of these formats:
+- Macos: .pkg
+- Windows: .msi, .exe
+- Linux: .deb
+
+Coming soon:
+- Ability to auto install from Apple VPP.
+- Install on iOS and iPadOS
+
+Using API:
+The same result can be achieved by using Fleet API.
+See our `POST /api/v1/fleet/software/package` API for details.
+Example:
```

Comment:
@lucasmrod, can you please provide an example call to our API for auto install?


../website/assets/images/articles/automatic-software-install-in-fleet-729x924@2x.png